app/models/solidus/gateway/braintree_gateway.rb

```diff
@@ -7,6 +7,7 @@ class Gateway::BraintreeGateway < ::Spree::Gateway
     preference :public_key, :string
     preference :private_key, :string
     preference :always_send_bill_address, :boolean, default: false
+    preference :verify_shipping_address, :boolean, default: true
```

Comment:
Perhaps `transmit_shipping_address` instead of verify? I believe that better represents what this preference is responsible for.